src/Aquifer.Jobs/Subscribers/ScoreResourceContentVersionSimilarityMessageSubscriber.cs

```diff
@@ -0,0 +1,246 @@
+using Aquifer.Common.Messages;
+using Aquifer.Common.Messages.Models;
+using Aquifer.Common.Tiptap;
+using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Azure.Storage.Queues.Models;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace Aquifer.Jobs.Subscribers;
+
+public sealed class ScoreResourceContentVersionSimilarityMessageSubscriber(
+    AquiferDbContext _dbContext,
+    ILogger<ScoreResourceContentVersionSimilarityMessageSubscriber> _logger)
+{
+    [Function(nameof(ScoreResourceContentVersionSimilarity))]
+    public async Task ScoreResourceContentVersionSimilarity(
+        [QueueTrigger(Queues.GenerateResourceContentSimilarityScore)]
+        QueueMessage queueMessage,
+        CancellationToken ct)
+    {
+        await queueMessage.ProcessAsync<ScoreResourceContentVersionSimilarityMessage, ScoreResourceContentVersionSimilarityMessageSubscriber>(
+            _logger,
+            nameof(ScoreResourceContentVersionSimilarity),
+            ProcessAsync,
+            ct);
+    }
+
+    private async Task ProcessAsync(QueueMessage queueMessage, ScoreResourceContentVersionSimilarityMessage message, CancellationToken ct)
+    {
+        _logger.LogInformation("Scoring machine translation...");
```

Comment:
I suggest adding more data from the `message` into the log so we know what exactly was being processed.